### PR TITLE
SAR-8763 Update read me files for ability to disable business verific…

### PR DESCRIPTION
**Pull request by woollett-r**
…ation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +26,12 @@ Creates a new journey, storing the journeyConfig against the journeyId.
 #### Request:
 Request body must contain the continueUrl and deskProServiceId fields. If nothing is provided for the optional service name, ```Entity Validation Service``` will be used.
 
+The field businessVerificationCheck enables calling services to bypass business verification. If the field is set to "false" and there is a successful match the entity will be registered. If not provided the default value used for this field is "true".
+
 ```
 {
   "continueUrl" : "/testUrl",
+  "businessVerificationCheck": false,
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply README.md:34]: Yes, when a request goes directly to the journey controller the default is to set the flag to "true" if it is not defined.

File: README.md [hunk 2]
```diff
@@ -32,21 +40,27 @@ Request body must contain the continueUrl and deskProServiceId fields. If nothin
...
 Creates a new journey for Ltd Company, storing the journeyConfig against the journeyId.
 #### Request:
 Request body must contain the continueUrl and deskProServiceId fields. If nothing is provided for the optional service name, ```Entity Validation Service``` will be used.
 
+The field businessVerificationCheck enables calling services to bypass business verification. If the field is set to "false" and there is a successful match the entity will be registered. If not provided the default value used for this field is "true".
+
 ```
 {
   "continueUrl" : "/testUrl",
+  "businessVerificationCheck": false,
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply README.md:63]: As above

File: README.md [hunk 3]
```diff
@@ -55,22 +69,27 @@ Request body must contain the continueUrl and deskProServiceId fields. If nothin
...
 Creates a new journey for Registered Society, storing the journeyConfig against the journeyId.
 #### Request:
 Request body must contain the continueUrl and deskProServiceId fields. If nothing is provided for the optional service name, ```Entity Validation Service``` will be used.
 
+The field businessVerificationCheck enables calling services to bypass business verification. If the field is set to "false" and there is a successful match the entity will be registered. If not provided the default value used for this field is "true".
+
 ```
 {
   "continueUrl" : "/testUrl",
+  "businessVerificationCheck": false,
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply README.md:92]: As. above.

File: README.md [hunk 4]
```diff
@@ -81,19 +100,24 @@ Status: **Created(201)**
...
 Creates a new journey for Charitable Incorporated Organisation, storing the journeyConfig against the journeyId.
 #### Request:
 Request body must contain the continueUrl and deskProServiceId fields. If nothing is provided for the optional service name, ```Entity Validation Service``` will be used.
 
+The field businessVerificationCheck enables calling services to bypass business verification. If the field is set to "false" and there is a successful match the entity will be registered. If not provided the default value used for this field is "true".
+
 ```
 {
   "continueUrl" : "/testUrl",
+  "businessVerificationCheck": false,
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply README.md:120]: Yes indeed.

File: TestReadme.md [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+    - Business Verification Stub
+2. Incorporated Entity Identification
+
+    - Get CT reference stub
+    - DES stub
+
+### GET test-only/create-limited-company-journey
+
+---
+This is a test entry point which simulates a service making the initial call to setup a create limited company journey.
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply TestReadme.md:19]: Updated

File: TestReadme.md [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+
+5. Business verification checkbox
+
+    - Used for skipping further verification checks carried out currently by Business Verification (SI)
+    - This is currently autofilled but can be changed
+
+### GET test-only/create-registered-society-journey
+
+---
+This is a test entry point which simulates a service making the initial call to setup a create registered society journey.
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply TestReadme.md:48]: Updated

File: TestReadme.md [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+
+5. Business verification checkbox
+
+     - Used for skipping further verification checks carried out currently by Business Verification (SI)
+     - This is currently autofilled but can be changed
+
+### GET test-only/create-cio-journey
+
+---
+This is a test entry point which simulates a service making the initial call to setup a create charitable incorporated organisation journey.
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply TestReadme.md:77]: Updated

File: TestReadme.md [hunk 8]
```diff
@@ -0,0 +1,260 @@
...
+5. Business verification checkbox
+
+      - Used for skipping further verification checks carried out currently by Business Verification (SI)
+      - This is currently autofilled but can be changed
+
+### GET test-only/create-journey
+#### Deprecated - use one of the specific journey urls instead (e.g. create-limited-company-journey)
+
+---
+This is a test entry point which simulates a service making the initial call to setup a journey.
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply TestReadme.md:107]: Updated

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (refaque-ahmad) [reply README.md:34]: Discussed it with Rich and agreed to set it to false

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (refaque-ahmad) [reply README.md:120]: Discussed it with Rich and agreed to set it to false

**Review by refaque-ahmad — APPROVED** [review]
LGTM

**Pull request merged**